kubetest/util.go

```diff
@@ -89,7 +89,19 @@ func getLatestClusterUpTime(gcloudJSON string) (time.Time, error) {
 // Pass in releasePrefix to get latest valid version of a specific release.
 // Empty releasePrefix means use latest across all available releases.
 func getLatestGKEVersion(project, zone, releasePrefix string) (string, error) {
-	res, err := control.Output(exec.Command("gcloud", "container", "get-server-config", fmt.Sprintf("--project=%v", project), fmt.Sprintf("--zone=%v", zone), "--format=value(validMasterVersions)"))
+	cmd := []string{
+		"container",
+		"get-server-config",
+		fmt.Sprintf("--project=%v", project),
+		"--format=value(validMasterVersions)",
+	}
+
+	// zone can be empty for regional cluster
+	if zone != "" {
+		cmd = append(cmd, fmt.Sprintf("--zone=%v", zone))
+	}
```

Comment:
This won't help unfortunately. We need two changes:

1. We need to pass "--region" flag (exactly one of flags --gcp-zone and --gce-region should be set at the kubetest level and that should be propagated here)

2. the "gke-command-group" flag needs to be honoured here


So ideally we should try reusing this:
https://github.com/kubernetes/test-infra/blob/master/kubetest/gke.go#L125
and
https://github.com/kubernetes/test-infra/blob/master/kubetest/gke.go#L150 + next if

Reply:
@wojtek-t I didn't see the --region flag though, maybe because that's only available in staging?